### PR TITLE
EZP-29335: In back office only last part of content URL uses translated name

### PR DESCRIPTION
**Pull request by ViniTou**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-29335](https://jira.ez.no/browse/EZP-29335)
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | `7.2`+
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

**TODO**:
- [x] Implement feature / fix a bug.
- [x] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

File: eZ/Publish/API/Repository/Tests/URLAliasServiceTest.php [hunk 1]
```diff
@@ -572,6 +573,81 @@ public function testListLocationAliases()
...
+     * Test for the listLocationAliases() method.
+     *
+     * @see \eZ\Publish\API\Repository\URLAliasService::listLocationAliases()
+     */
+    public function testListLocationAliasesWithTranslations()
+    {
+        $repository = $this->getRepository();
+
+        /* BEGIN: Use Case */
+        // $locationId contains the ID of an existing Location
```
Comment (andrerom) [inline eZ/Publish/API/Repository/Tests/URLAliasServiceTest.php]: ..

**Comment by andrerom** [comment]
`extractPath()` is also used by:
```
public function load($id)  
public function listGlobalAliases($languageCode = null, $offset = 0, $limit = -1)
```

So this seems to indicate their behavior is changed as well => Which makes me think we don't have enough integration test coverage here _(as nothing is failing)_.

**Comment by ViniTou** [comment]
@andrerom 
Totally unserstood, take into consideration that urls before that change are still valid (and new ones were valid before that change) so we can be pretty safe with it?

If not, what do you suggest?

**Comment by andrerom** [comment]
@alongosz What do you think? Should we try to add some more coverage here for url alias on the affected api's?

File: eZ/Publish/API/Repository/Tests/URLAliasServiceTest.php [hunk 2]
```diff
@@ -572,6 +573,80 @@ public function testListLocationAliases()
         return array($loadedAliases, $location);
     }
 
+    /**
+     * Test for the listLocationAliases() method.
+     *
+     * @see \eZ\Publish\API\Repository\URLAliasService::listLocationAliases()
+     */
+    public function testListLocationAliasesWithTranslations()
```
Comment (alongosz) [inline eZ/Publish/API/Repository/Tests/URLAliasServiceTest.php]: In this case rather `...WithPrioritizedLanguagesList`. We already have `testListLocationAliasesWithLanguageCodeFilter`, so `WithTranslation` is a little bit ambiguous here.

**Comment by alongosz** [comment]
@ViniTou changed base to `7.2`. Please rebase after applying changes.

**Comment by ViniTou** [comment]
This seems to be unsolvable without bc breaks as prioritized languages are more important in URL alias generation than content language.

**Pull request closed without merging**